cli/medperf/logging/formatters/newline_formatter.py

```diff
@@ -0,0 +1,21 @@
+import logging
+
+
+class NewLineFormatter(logging.Formatter):
+    def __init__(self, fmt, datefmt=None):
+        """
+        Init given the log line format and date format
+        """
+        logging.Formatter.__init__(self, fmt, datefmt)
+
+    def format(self, record):
+        """
+        Override format function
+        """
+        msg = logging.Formatter.format(self, record)
+
+        if record.message != "":
+            parts = msg.split(record.message)
+            msg = msg.replace("\n", "\n" + parts[0])
+
+        return msg
```

Comment:
why are we overriding`__init__`?
Also, in both `__init__` and `format`, is `logging.Formatter` the same as `super()`?

Reply:
You're right. This was taken directly from a Stack Overflow thread, but could be improved by the points you raised. I'll modify the code to remove or simplify unnecessary sections.